### PR TITLE
feat(linter): eslint-plugin-jsx-a11y no-autofocus 

### PR DESCRIPTION
**Pull request by msdlisper**
impl linter for #1141

### CODE REVIEW

File: crates/oxc_linter/src/rules/jsx_a11y/no_autofocus.rs [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+use crate::{context::LintContext, rule::Rule, utils::has_jsx_prop, AstNode};
+
+const HTML_TAG: [&str; 149] = [
```
Comment (IWANABETHATGUY) [inline crates/oxc_linter/src/rules/jsx_a11y/no_autofocus.rs:11]: try use `phf::set!`

Comment (msdlisper) [reply crates/oxc_linter/src/rules/jsx_a11y/no_autofocus.rs:11]: I try

**Comment by camc314** [comment]
> Why does my linter affect the Vue.js semantic analysis? 🤣
> 
> ```
> semantic[vue.js] Regression
> ```

i think it's a buggy test case, there's often 5% regression/improvement on PRs

File: crates/oxc_linter/src/rules/jsx_a11y/no_autofocus.rs [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    "ul",
+    "var",
+    "video",
+    "wbr",
+    "xmp",
+};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("eslint-plugin-jsx-a11y(no-autofocus): The `autofocus` attribute is found here")]
+#[diagnostic(severity(warning), help("Remove `autofocus` attribute"))]
```
Comment (camc314) [inline crates/oxc_linter/src/rules/jsx_a11y/no_autofocus.rs:167]: can we explain why this is bad?

**Pull request merged**